### DISCUSSION TITLE
Issue with challeging ngrok url on facebook developers

### DISCUSSION THREAD

**Discussion by rootkit7628** [iTeam-S/Ampalibe#52] [General]
If you encounter some issues with challenging Facebook Developers, you should know that Facebook doesn't allow ngrok url configured with `europe - eu` as region so you need to move it to `us` like this :
👉🏿   `ngrok http <port> --region us`

**Comment by gaetan1903** [comment]
I encountered the same errors too but with a vps in the europe region.


Facebook does not like all European servers, it may be because of the GDPR which is applied in Europe.
